css/app-sidebar.scss

```diff
@@ -83,22 +83,30 @@
 		display: flex;
 		flex-direction: column;
 		justify-content: space-between;
+		overflow: unset !important;
+		max-height: unset !important;
 
 		&__buttons {
 			padding-top: 14px;
-			position: absolute;
+			position: fixed;
 			bottom: 0px;
 			z-index: 2;
-			width: 97%;
-			left: 0px;
+			width: calc(27vw - 10px);
+			min-width: 300px - 10px;
+			max-width: 500px - 10px;
```

Comment:
```suggestion
			width: calc(27vw - 11px);
			min-width: 300px - 11px;
			max-width: 500px - 11px;
```

The border is overlapped by 1 pixel.

![sidebar-border-pixel](https://user-images.githubusercontent.com/1479486/127320795-6b8be648-133a-457d-9b80-ac80c6dca19c.png)

Reply:
good find! Fixed :)